GH Configuration: Added some more details to the PR Template comments. (#4843)
awsdocs/aws-doc-sdk-examples: .github/PULL_REQUEST_TEMPLATE.md

<!--

> Thank you for making a submission to the *aws-doc-sdk-examples* repository. Please provide an overview of the intended changes from this PR.

Reviewer: ```suggestion
Thank you for making a submission to the *aws-doc-sdk-examples* repository. Briefly tell us what you intend to change with this PR.
```

> PR Titles should be similar to: <Language>: <what you did in> <service>
> PR Bodies should be descriptive of interesting things; however, with a good title, the body isn't necessary and can be deleted.

Reviewer: ```suggestion
Format the PR _title_ like this: <Language>: <what you did in> <AWS service>
In the PR _body_, you can describe your changes in more detail. If the title is descriptive enough, however, you can delete the body.
```
